.github/workflows/flaky-test-detector.yml

```
@@ -1,123 +1,123 @@
-
-      - name: Upload flaky test artifacts
-        if: always()
-        uses: actions/upload-artifact@v6
-        with:
-          name: flaky-test-results
-          path: |
-            flaky-tests.json
-            flaky-report.md
-            playwright-report/
-          retention-days: 30
+# name: Flaky Test Detector
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)